05_assessing_code_quality/1_project_structure.md

```diff
@@ -0,0 +1,111 @@
+# Project Structure
+
+## What is software design
+
+The goal of software architecture is to minimize the human resource required to build and maintain software systems. If the effort required to maintain a system grows with time, then the design is bad. If it's constant or shrinks with time, then the design is bad.
```

Comment:
[change] I am not sure that the last sentence is correct. If effort increases it's bad. If it stays constant or shrinks is good

Reply:
```suggestion
The goal of software architecture is to minimize the human resources required to build and maintain software systems. If the effort required to maintain a system grows with time, then the design is bad. If it's constant or shrinks with time, then the design is good.
```

Ops!